src/components/ConversationIcon.js

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import Avatar from 'react-avatar';
+import PropTypes from 'prop-types';
+class ConversationIcon extends PureComponent {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {
+      conversation,
+      className,
+      extraContent,
+      extraContentArguments,
+      reactAvatarProps
+    } = this.props;
+    
+    const { title, profileImageUrl } = conversation;
+    const hasProfileImage = profileImageUrl && profileImageUrl.trim();
+    const defaultAvatarProps = {
+      maxInitials: 2,
+      size: 40,
+      round: true
+    }
+
+    const finalReactAvatarProps = { ...defaultAvatarProps, ...reactAvatarProps };
+
+    return (
+      <React.Fragment>
+        {
+          !hasProfileImage &&
+          <div title={title} className= {className} style={{ backgroundImage: `url(${profileImageUrl})` }}>
+            <Avatar name={title} {...finalReactAvatarProps} />
+            {extraContent && extraContent(...extraContentArguments)}          
+          </div>
+        }
+        {
+          hasProfileImage &&
+          <div title={title} className= {className} style={{ backgroundImage: `url(${profileImageUrl})` }}>
+            {extraContent && extraContent(...extraContentArguments)}
```

Comment:
I think that this should also be possible by `Avatar` component. Please check it.

Reply:
Done!